src/style.css

```diff
@@ -1,3 +1,55 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
+  background-color: wheat;
   margin: 0;
 }
+
+.header {
+  padding: 0 20px;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  padding: 20px 0;
+}
+
+.logo {
+  display: block;
+}
+
+.nav_link {
+  margin: 15px;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav_link:hover {
+  color: #d927e9;
+}
+
+.nav_link4 {
+  text-decoration: none;
+  color: #00acdc;
+}
+
+.is-active {
+  text-decoration: none;
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: -20px;
```

Comment:
The `bottom` property in the `.is-active::after` pseudo-element is missing a unit. Please specify a unit (e.g., `px`, `em`, etc.) to ensure proper positioning of the pseudo-element.